README.md

```diff
@@ -239,14 +239,118 @@ makes it very easy to compare edits.
 
 1. PR makes it easy to compare difference and start discussions. To issue a PR through github web page:
 
-   ![alt text](https://github.com/chiahaoliu/group_git_workflow/blob/doc_gitworkflow/img/issue_PR.png "")
+   ![alt text](https://github.com/Billingegroup/group_git_workflow/blob/doc_gitworkflow/img/issue_PR.png "")
 
-   ![alt text](https://github.com/chiahaoliu/group_git_workflow/blob/doc_gitworkflow/img/create_PR.png "")
+   ![alt text](https://github.com/Billingegroup/group_git_workflow/blob/doc_gitworkflow/img/create_PR.png "")
 
 1. After issuing a PR, other developers can view your edits, add comment and eventually decide whether to pull in your PR.
 1. You can always update your PR by pushing new edits to the same branch under your fork.
 
-# Step 7: Merge after PR is pulled in
+
+# Step 7: Rebase
+Sometimes you may find `github` shows `This branch has conflicts that
+must be resolved` in your PR page. This simply means the remote branch this PR is based on is in a different working history as `upstream\master`. `github` provides a button in the PR page to solve the conflict, but **DO NOT** use this approach since it creates an extra commit and could further divert you away from `upstream\master`. For this situation, the step you need is **rebase**. Here are steps of rebase with our good friend `PyCharm`.
+
```

Comment:
please change the last sentence to: 
"rebasing can be carried out on the command-line using git commands (described later) but it is much easier using a python IDE with built-in rebasing tools, such as our good friend `PyCharm`.  Here we reproduce the steps to do the rebase using PyCharm.  It will be slightly different if you are using a different IDE.  Look lower down if you want to do it directly with Git."